src/components/ContactPreview.tsx

```diff
@@ -12,12 +12,39 @@ type Props = {
   contact: Profile
 }
 
+export function useStatus() {
+  return {
+    isUserOnline: (contact: Profile) =>
+      contact?.timestamp_last_connection
```

Comment:
added parenthesis here `(Date.parse(new Date().toISOString()) -Date.parse(contact.timestamp_last_connection)) / 1000 < 60` this is why unit tests exist 😂